src/components/Authenticate.js

```diff
@@ -0,0 +1,5 @@
+const devAuth = {
```

Comment:
Improve expression of intent: Authenticate.js and devAuth are not meaningful names. Maybe just AuthVars and you can change to a variable (let isAuthenticated = false).  Might be better to use cookies to store log in state, not sure.  See https://stackoverflow.com/questions/40625918/checking-authentication-in-react/40627533